napari/layers/image/image.py

```diff
@@ -282,9 +282,14 @@ def _update_thumbnail(self):
             self._thumbnail_shape[:2], image.shape[:2]
         ).min()
         if self.multichannel:
-            downsampled = ndi.zoom(
-                image, (zoom_factor, zoom_factor, 1), prefilter=False, order=0
-            )
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
Just fyi, I fixed this in scipy master just a couple of days ago. ;)

Reply:
https://github.com/scipy/scipy/pull/10396

Reply:
comment added about removal in `0.14`